### ISSUE TITLE
Add support for Indipop song category

### ISSUE THREAD

**Issue by amulyagaur** [amulyagaur/EasyMP3#13]
Currently only bollywood songs are supported. We can extend it to include indipop category as well which is available at this url : https://downloadming3.com/category/indipop-mp3-songs

**Comment by Deshdeepak1** [comment]
I would like to work on this.

**Comment by Deshdeepak1** [comment]
Should I use argument for it, like --indipop while running the program?

**Comment by amulyagaur** [comment]
> Should I use argument for it, like --indipop while running the program?

Yes.. can be done this way.. we just have to inform the user about this in the readme.
Otherwise showing option on runtime is more intuitive according to me. Let me know your thoughts

**Comment by Deshdeepak1** [comment]
I think the program should be like if user directly starts the program, it should show 1st page of bollywood-mp3-songs , 
if it adds --indipop option it should show 1st page of indipop. If -h, --help it shows usageand options. Readme should be updated to with basic help info.

**Comment by amulyagaur** [comment]
Cool.. seems fine
